src/leo-server-app/src/routes/schedule.ts

```diff
@@ -388,21 +389,36 @@ router.get(
 router.get(
   "/getScheduleBySatelliteAndTime",
   async (req: GetScheduleBySatelliteAndTimeProp, res: any) => {
-    const { satelliteId, status = ScheduleStatus.FUTURE, time } = req.query;
+    const {
+      satelliteId,
+      status = ScheduleStatus.FUTURE,
+      startTime = new Date().setDate(new Date().getDate()),
+      endTime = ((startTime: Date) => {
+        let endDate = new Date(startTime);
+        endDate.setDate(endDate.getDate() + 14);
+        return endDate;
+      })(new Date(startTime)),
+    } = req.query;
 
-    const convertedTime = new Date(time);
+    if (!mongoose.isValidObjectId(satelliteId)) {
+      return res.status(500).json({ error: "Invalid ID" });
+    }
 
+    const convertedStartTime = new Date(startTime);
+    convertedStartTime.setHours(convertedStartTime.getHours() + 5);
+    const convertedEndTime = new Date(endTime);
+    convertedEndTime.setHours(convertedEndTime.getHours() + 29);
```

Comment:
What is the +5 and +14 doing?

Reply:
changed back to original but mongo needs to be updated